centreon/packages/ui/src/Listing/ActionBar/index.tsx

```diff
@@ -130,7 +184,7 @@ const MemoListingActionBar = ({
               colSpan={3}
               count={totalRows}
               labelDisplayedRows={labelDisplayedRows}
-              labelRowsPerPage={t(labelRowsPerPage)}
+              labelRowsPerPage={null}
```

Comment:
Be careful, a lot of tests depend on this label.

Reply:
the ticket mentions that we must remove the label to have more space, I corrected the failed tests captured by the ci